src/components/Media/Image/Image.tsx

```diff
@@ -78,7 +80,7 @@ const Image = (props: ImageAllProps) => {
     const imageBackground = (oneImage: ImageProps) => {
         const imageData = getMediaImage(oneImage);
         return (
-            <animated.div style={{transform: parallaxInterpolate}}>
+            <animated.div style={{transform: parallaxInterpolate}} data-qa={qaIdOfAnimatedDiv}>
```

Comment:
Don't we need custom qa here too?

Reply:
Now, it seems to me that we do not need custom ones
I cannot see necessity to pass a bunch of qaIds so deep in a component.

What we can do here is add a custom role to <animated.div />

```ts
<animated.div role="animated-div" />
```
By this way in a test we can access this element like
```ts
screen.getByRole('animated-div') // instead of screen.getByTestId(qaIdOfAnimatedDiv)
```

Reply:
UPDATE: Have talk to Andrey. Adding role is not good idea. Describe it test's readme